packages/rabid/PKGBUILD

```diff
@@ -0,0 +1,32 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=rabid
+_gemname=rabid
+pkgver=0.0.1
+pkgrel=1
+pkgdesc='A CLI tool and library allowing to simply decode all kind of BigIP cookies.'
+arch=('any')
```

Comment:
groups=() is missing here. add 'blackarch-webapp' and 'blackarch-misc' in this case please